portfolio/src/main/webapp/css/abstracts/grid.css

```diff
@@ -0,0 +1,36 @@
+/*
+    This file is for the grid system on the page.
+    Use these for laying objects next to / on top of each other
+*/
+
+.row {
+    height: 100%;
+}
+
+.row:after {
+    clear: both;
+    content: '';
+    display: table;
+}
+
+.row:not(:last-child) {
+    margin: 0 0 var(--gutter-vertical) 0;
+}
+
+.col-1-of-3 {
+    float: left;
+    width: calc((100% - var(--gutter-horizontal)*2)/3);
+}
+
+.col-1-of-3:not(:last-child) {
+    margin: 0 var(--gutter-horizontal) 0 0;
+}
+
+.col-2-of-3 {
```

Comment:
Is there a col-3-of-3, or am I missing something? The naming suggests this is part of a 3-column layout.

Reply:
A column 3 of 3 would end up just being a row. Naming is more like a fraction - this column takes up 2 columns of 3 available, etc. Adding some comments in the grid file to explain the convention - thanks!